specs.json

```
@@ -237,6 +237,8 @@
       ]
     }
   },
+  "https://screen-share.github.io/captured-mouse-events/",
+  "https://screen-share.github.io/element-capture/",
```

Need to be explicit about status for now because of #944:

```suggestion
  {
    "url": "https://screen-share.github.io/captured-mouse-events/",
    "nightly": {
      "status": "Draft Community Group Report"
    }
  },
  {
    "url": "https://screen-share.github.io/element-capture/",
    "nightly": {
      "status": "Draft Community Group Report"
    }
  },
```

---

Thanks!